designs/virtualization/4.x/vnc-in-new-window/vnc-in-new-window.md

```diff
@@ -0,0 +1,47 @@
+---
+parent: Virtualization
+version: 4.x
```

Comment:
I know this work is being implemented now. Can we confirm it isn't getting into 4.6?